docs/smtchecker.rst

```diff
@@ -489,6 +517,7 @@ was not proven is passed over to BMC. You can choose a specific engine via the C
 option ``--model-checker-engine {all,bmc,chc,none}`` or the JSON option
 ``settings.modelChecker.engine={all,bmc,chc,none}``.
 
+
```

Comment:
You intentionally added this empty line?

Reply:
I think I wrote some stuff there, removed it, and the blank line stayed, Will remove it.